### DISCUSSION TITLE
Welcome to ldap-self-service Discussions!

### DISCUSSION THREAD

**Discussion by alvinsiew** [alvinsiew/ldap-self-service#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by walkerdgp** [comment]
I have this error, when i try to change password

POST request successful
Status = exit status 255

**Comment by Rabidurbian** [comment]
Hi. I have erro as WALKERDGP.
How configuration for config.yml file should looks like ? Because no matter what I've type in this file, nothing is working. My OpenLDAP base is operative, I may connect to it from different applications like CheckMK or LDAP explorer.
